build.ps1

```diff
@@ -0,0 +1,47 @@
+$ClientImportPath="github.com/hazelcast/hazelcast-go-client"
+$ListOfPackages = New-Object System.Collections.ArrayList
+go list $ClientImportPath/... | Select-String -pattern ".*/test|.*/compatibility|.*/rc|.*/sample" -notmatch | %{$ListOfPackages.add($_)}
+$PackageList=$ListOfPackages -join ","
+
+#run linter
+cd $env:gopath\src\$ClientImportPath
+.\linter.ps1
+
+# currently comment out due to a lot of found issues
+#If ($lastExitCode -ne 0) {
+#    Exit 1
+#}
+
+If (Test-Path -PathType Container "$env:gopath\src\github.com\apache\thrift\") {
+    Write-Host "thrift already exists, not downloading."
+} Else {
+    go get github.com/apache/thrift/lib/go/thrift
+    cd $env:gopath\src\github.com\apache\thrift
+    git fetch --tags --quiet
+    git checkout 0.10.0    
+}
+
+cd $env:gopath\src\$ClientImportPath
+go build
+
+cd $env:gopath
+.\start-rc.ps1
+
+Start-Sleep -s 11
+
+# Run vet tools (Compiler warning plugin)
+go vet $ClientImportPath > vet.txt
```

Comment:
This is actually not necessary, we already do vet analysis with go metalinter. I will remove this from `build.sh` as well.

Reply:
Ok, thanks, I will remove it